oml/utils/download_mock_dataset.py

```diff
@@ -188,4 +192,56 @@ def get_mock_texts_dataset() -> Tuple[pd.DataFrame, pd.DataFrame]:
     return df_train, df_val
 
 
-__all__ = ["get_mock_images_dataset", "get_mock_texts_dataset"]
+def get_mock_audios_dataset(
```

Comment:
seems like there is almost no difference between `get_mock_images_dataset` and `get_mock_texts_dataset`

let's introduce a function `_download_dataset` implementing shared logic and reuse it in both `get_mock_images_dataset` and  `get_mock_texts_dataset`?

Reply:
done